### PR TITLE
[QA-337] : Updated IPV core scripts to optimise load injector memory utilisation

### PR DESCRIPTION
**Pull request by abhinaysrivastavaperf**
### What?
Updated IPV core scripts to reduce load injector memory utilisation

#### Changes:
- Commented the code for getStaticResources function (including import module).
- Reduced think times.
- Reduced max virtual users to 3000. 

---

### Why?
These changes are expected to reduce load injector memory utilisation which is currently a bottleneck for running IPV core stress tests.

### CODE REVIEW

**Review by sachadorf13 — APPROVED** [review]

File: deploy/scripts/src/ipv-core/test.ts [hunk 1]
```diff
@@ -122,20 +122,20 @@ export function passport (): void {
...
+      // if (env.staticResources) getStaticResources(response)
       return response
     },
     {
       'is status 200': (r) => r.status === 200,
       'verify page content': (r) => (r.body as string).includes('Tell us if you have one of the following types of photo ID')
     }))
 
-  sleep(Math.random() * 2)
+  sleep(Math.random() * 1)
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:138]:
```suggestion
  sleep(Math.random())
```
Well we can drop all these multiplications by `1`

Should rebase off the changes of #330 anyway

**Review by Tom-Dann — CHANGES_REQUESTED** [review]

File: deploy/scripts/src/ipv-core/test.ts (hunk 1 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Ah didn't pay attention that I was reducing it to 1. Apologies.
Have now rebased this branch so all updated.

File: deploy/scripts/src/ipv-core/test.ts [hunk 2]
```diff
@@ -363,12 +363,14 @@ export function drivingLicence (): void {
...
     res = timeRequest(() => {
       const response = http.get(env.ipvCoreURL + res.headers.Location, { tags: { name: 'B02_DrivingLicence_04_DCMAWContinue_02_CoreCall' } })
-      if (env.staticResources) getStaticResources(response)
+      // if (env.staticResources) getStaticResources(response)
       return response
     },
     { isStatusCode200, ...pageContentCheck('Enter your UK photocard driving licence details and answer security questions online') })
   })
 
+  sleepBetween(1, 3)
```
Comment (Tom-Dann) [inline deploy/scripts/src/ipv-core/test.ts:372]:
```suggestion
  sleepBetween(0.5, 1)
```
Let's reduce the think times to reduce iteration time and hence reduce VU count and memory usage

**Review by Tom-Dann — CHANGES_REQUESTED** [review]

File: deploy/scripts/src/ipv-core/test.ts (hunk 2 above)
Comment (abhinaysrivastavaperf) [reply deploy/scripts/src/ipv-core/test.ts]: Done

**Review by Tom-Dann — APPROVED** [review]

**Pull request merged**